src/index.html

```diff
@@ -7,8 +7,112 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fronts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet">
   </head>
+
   <body>
+
     <h1>Moyo header</h1>
+
+    <header class="header" id="home">
+
+      <a href="#home" class="logo__link">
```

Comment:
looks like logo is an element of header

```suggestion
      <a href="#home" class="header__logo">
```